### ISSUE TITLE
Idea: mark a Slack thread as one to automatically add to the timeline

### ISSUE THREAD

**Issue by reidmix** [heroku/retrodot#251]
In incident 1090, a thread was used to communicate updates from the hangout, which was a good source of what was happening for the incident in general.  [Link to the thread](https://heroku.slack.com/archives/C1NV1FUG1/p1491321800432217)

**Comment by lexelby** [comment]
While it would have been useful in this specific incident, do you think this is really generally applicable?  Even with this thread, there was still a certain amount of noise, since Peter was giving a running transcript of a live conversation.  Seems like it wouldn't be too hard to just react to interesting messages with the emoji instead?

**Comment by reidmix** [comment]
I'm starting to think that what keeps coming up and perhaps why the thread seemed so interesting to me is the desire for incident-specific Slack rooms.   I know we just had a discussion about this on #SRE and it comes up for concurrent incidents, but the desire to have this came up in this incident too.  Personally, I tend to like #platform-incidents as it really does serve our needs and keeps things centralized... in Slack it's much easier than HipChat to find convos in it -- with that being said, I recently went into the secops room for an incident and that was rather nice as well.  So I don't have a definitive answer.

**Comment by lexelby** [comment]
Yeah.  I'm kind of on the fence.  On the one hand, the discoverability is pretty useful, and people sometimes check in and then have a useful insight that no one else on the room has.  I literally just did that, in fact. :)

On the other hand, it's a lot tidier to have one room per incident.  Slack is easier, but I still find it painful to read through the long archive in #platform-incidents.

Perhaps if we spun off a room but retrodot posted the timeline to #p-i live as it's added to?

**Comment by reidmix** [comment]
That's a super interesting idea!
